src/common/containers/builds.js

```diff
@@ -4,14 +4,25 @@ import Helmet from 'react-helmet';
 
 import BuildHistory from '../components/build-history';
 import { Message } from '../components/forms';
+import Spinner from '../components/spinner';
+
 import { fetchSnap } from '../actions/snap-builds';
 
 import styles from './container.css';
 
 class Builds extends Component {
+  fetchInterval = null
 
   componentWillMount() {
     this.props.dispatch(fetchSnap(this.props.fullName));
+
+    this.fetchInterval = setInterval(() => {
+      this.props.dispatch(fetchSnap(this.props.fullName));
+    }, 5000);
```

Comment:
LP's buildd-manager cycle is 15 seconds, so there's no point in making requests any more frequently than that as they can't possibly have new data.  15 seconds should be OK.